redhat/rdma.conf

```diff
@@ -1,18 +1,27 @@
-# Load IPoIB
-IPOIB_LOAD=yes
-# Load SRP (SCSI Remote Protocol initiator support) module
-SRP_LOAD=yes
-# Load SRPT (SCSI Remote Protocol target support) module
-SRPT_LOAD=yes
-# Load iSER (iSCSI over RDMA initiator support) module
-ISER_LOAD=yes
-# Load iSERT (iSCSI over RDMA target support) module
-ISERT_LOAD=yes
-# Load RDS (Reliable Datagram Service) network protocol
-RDS_LOAD=no
-# Load NFSoRDMA client transport module
-XPRTRDMA_LOAD=yes
-# Load NFSoRDMA server transport module
-SVCRDMA_LOAD=no
-# Load Tech Preview device driver modules
-TECH_PREVIEW_LOAD=no
+# These modules are loaded by the system if any RDMA devices is installed
```

Comment:
Why do you copy this file from kernel-boot/ to redhat/?

Reply:
For RHEL system with RDMA device, we always load ib_isert, ib_srpt and ib_umad modules. But they are unavailable in kernel-boot/modules/rdma.conf .

Reply:
IB umad is already autoloaded properly only for IB and OPA, not needed in other cases.